metaflow/plugins/project_decorator.py

```diff
@@ -17,7 +17,7 @@ class ProjectDecorator(FlowDecorator):
     Specifies what flows belong to the same project.
 
     A project-specific namespace is created for all flows that
-    use the same `@project(name)`.
```

Comment:
nit: `my-project` is actually not a valid project name :)

Reply:
Thanks, nice catch! Definitely important thing in this case. Updated.